pages/library/_viewId.vue

```diff
@@ -29,7 +29,7 @@
       <filter-button
         v-if="isSortable"
         :collection-info="collectionInfo"
-        :disabled="loading"
+        :disabled="loading || !(items.length && !hasFilters)"
```

Comment:
The logic here is wrong. If there are no items, it'll always be disabled. So you won't be able to clear the filters. I think it should be:
```suggestion
        :disabled="loading || (!items.length && !hasFilters)"
```

Reply:
Ah good catch! I also noticed that hasFilters was using `every` instead of `some`